incuna-sass/_incuna-normalize.sass

```diff
@@ -0,0 +1,13 @@
+// Basic styles that we end up adding to all of our projects
+
+body
+    *
+        box-sizing: border-box
+        line-height: 1.4
```

Comment:
I don't think you should set a line-height on all elements like this, just set it on the `body` as other elements will inherit it.